x/wasm-storage/keeper/keeper.go

```diff
@@ -148,11 +148,11 @@ func (k Keeper) IterateExecutorWasms(ctx sdk.Context, callback func(wasm types.E
 	return nil
 }
 
-// ListDataRequestWasms returns hashes and expiration block heights
-// of all data request wasms in the store.
-func (k Keeper) ListDataRequestWasms(ctx sdk.Context) []string {
+// ListOraclePrograms returns hashes and expiration block heights
+// of all oracle programs in the store.
+func (k Keeper) ListOraclePrograms(ctx sdk.Context) []string {
```

Comment:
Should we add pagination here? Given that we have to load the bytecode for all Oracle Programs I imagine the load might increase quite rapidly.

Reply:
good idea.. done